src/ORM/DataObjectSchema.php

```diff
@@ -151,7 +151,7 @@ public function sqlColumnForField($class, $field, $tablePrefix = null)
      *
      * @param string $class
      *
-     * @return string Returns the table name, or null if there is no table
+     * @return string|null Returns the table name, or null if there is no table
```

Comment:
The method already returns null, so this is just setting the correct type in the PHPDoc